frontend/src/scenes/insights/Insights.js

```diff
@@ -109,183 +108,187 @@ function _Insights() {
-                                    headStyle={{ backgroundColor: 'rgba(0,0,0,.03)' }}
-                                >
+                                        )}
+
+                                        {showDateFilter[activeView] && <DateFilter disabled={dateFilterDisabled} />}
+
+                                        {showComparePrevious[activeView] && <CompareFilter />}
+                                        <SaveToDashboard
+                                            item={{
+                                                entity: {
+                                                    filters: allFilters,
+                                                    annotations: annotationsToCreate,
+                                                },
+                                            }}
+                                        />
+                                    </div>
+                                }
+                                headStyle={{ backgroundColor: 'rgba(0,0,0,.03)' }}
+                            >
+                                <div>
                                     {showErrorMessage ? (
                                         <ErrorMessage />
-                                    ) : showTimeoutMessage ? (
-                                        <TimeOut isLoading={isLoading} />
-                                    ) : featureFlags['remove-shownas'] ? (
-                                        {
-                                            [`${ViewType.TRENDS}`]: <TrendInsight view={ViewType.TRENDS} />,
-                                            [`${ViewType.STICKINESS}`]: <TrendInsight view={ViewType.STICKINESS} />,
-                                            [`${ViewType.LIFECYCLE}`]: <TrendInsight view={ViewType.LIFECYCLE} />,
-                                            [`${ViewType.SESSIONS}`]: <TrendInsight view={ViewType.SESSIONS} />,
-                                            [`${ViewType.FUNNELS}`]: <FunnelInsight />,
-                                            [`${ViewType.RETENTION}`]: <RetentionContainer />,
-                                            [`${ViewType.PATHS}`]: <Paths />,
-                                        }[activeView]
                                     ) : (
-                                        {
-                                            [`${ViewType.TRENDS}`]: <TrendInsight view={ViewType.TRENDS} />,
-                                            [`${ViewType.SESSIONS}`]: <TrendInsight view={ViewType.SESSIONS} />,
-                                            [`${ViewType.FUNNELS}`]: <FunnelInsight />,
-                                            [`${ViewType.RETENTION}`]: <RetentionContainer />,
-                                            [`${ViewType.PATHS}`]: <Paths />,
-                                        }[activeView]
+                                        showTimeoutMessage && <TimeOut isLoading={isLoading} />
```

Comment:
Is something weird going on here or is this master merge weirness?

I think this block got doubled (view the diff with https://github.com/PostHog/posthog/pull/3278/files?w=1# to see what I mean)

Reply:
should be fixed now. the only chang i wanted to make inthat file was removing the user.team condition